### PR TITLE
chore: empty config test [dont merge]

### PR DESCRIPTION
**Pull request by jpvajda**
this a test to see how we handle an empty config related to this issue:

https://github.com/newrelic/newrelic-quickstarts/issues/633

### CODE REVIEW

File: quickstarts/empty-config-test/config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+# this is an empty file
```
Comment (aswanson-nr) [inline quickstarts/empty-config-test/config.yml:1]:
```suggestion
```

**Comment by aswanson-nr** [comment]
@jpvajda I deleted the comment in the `config.yml` just to be sure

**Comment by jpvajda** [comment]
@aswanson-nr the jobs aren't running, and just queued, but I think that is related to the Github issues we've been seeing all day.

**Comment by jpvajda** [comment]
@aswanson-nr nice! It seems we now have a very clear error in this case for a empty config, so I think we can  close out #633 unless you disagree. 

```
 Error: quickstarts/empty-config-test/config.yml
	 '#/type' must be object
```

**Comment by zstix** [comment]
We have confirmed what was necessary for this PR. Will close this.

**Pull request closed without merging**